cdk/index.ts

```diff
@@ -56,7 +56,7 @@ if (!util.isString(environmentParam)) {
 const environmentName = util.normalizeEnvironmentName(environmentParam);
 const environmentId = util.lowerCaseEnvironmentId(environmentParam);
 
-const spaStack = new StaticSiteStack(app, environmentName + "StaticSpa", {
+new StaticSiteStack(app, environmentName + "StaticSpa", {
```

Comment:
**Context**: With `const spaStack =` TypeScript is mad. Without it, SonarCloud is mad. I've dismissed SonarCloud's complaint for now because I'd rather have code that compiles than code that is "correct". Unfortunately, the entire way that the CDK works is through side effects of constructors. Perhaps there are arguments to be made that isn't ideal but it'll be easier to dismiss complaints from SonarCloud as false positives than to convince the CDK to totally refactor itself :)

Reply:
So that's why SonarCloud hates our stack definition file.